crates/autopilot/src/database/onchain_order_events.rs

```diff
@@ -3,7 +3,7 @@ pub mod event_retriever;
 
 use super::{
     events::{bytes_to_order_uid, meta_to_event_index},
-    Metrics, Postgres,
+    Metrics as DatabaseMetrics, Postgres,
```

Comment:
Just a style thing (so 100% feel free to ignore), but here you can:
```rust
use crate::database;
```

Then you can reference it as `database::Metrics` instead of `DatabaseMetrics`.

Reply:
I prefer not to because `database` already represents the `database` crate in this file, while the metrics come from `autopilot::database` so there would be some extra work needed to disambiguate the two that imho isn't improving readability.